cluster/node-pools/master-default/userdata.yaml

```diff
@@ -214,7 +214,7 @@ write_files:
             requests:
               cpu: 100m
               memory: 200Mi
-        - image: registry.opensource.zalan.do/teapot/admission-controller:master-91
+        - image: registry.opensource.zalan.do/teapot/admission-controller:master-89
```

Comment:
This is wrong.

Reply:
Fixed manually

Reply:
I did the merge conflict resolution wrong (because I didn't realize the revert). Let's delete this and have the bot recreate a clean branch.

Reply:
New PR without any conflicts: https://github.com/zalando-incubator/kubernetes-on-aws/pull/3925